mobile/src/main/java/org/openhab/habdroid/ui/AbstractItemPickerActivity.kt

```diff
@@ -240,6 +245,22 @@ abstract class AbstractItemPickerActivity : AbstractBaseActivity(), SwipeRefresh
                 return@launch
             }
 
+            if (connection is DemoConnection) {
+                Log.d(TAG, "Demo mode is on => Show snackbar")
+                val snackbar = Snackbar.make(
+                    findViewById(R.id.activity_content),
+                    R.string.info_demo_mode_short,
+                    Snackbar.LENGTH_INDEFINITE
+                )
+                snackbar.setAction(R.string.turn_off) {
+                    applicationContext.getPrefs().edit {
```

Comment:
No need for using `applicationContext` here (can just omit is, since `this@AbstractItemPickerActivity` is a context as well)